remove the block fill rate limit of 70% when farming a block

arvidn opened the pull request:
### Purpose:

to increase transaction rate

### Current Behavior:

Farmers only fill blocks to 70% full

### New Behavior:

Farmers fill blocks all the way. But blocks are compressed and have potential to fit more transactions.

### Testing Notes:

<!-- Attach any visual examples, or supporting evidence (attach any .gif/video/console output below) -->

arvidn committed:
remove the block fill rate limit of 70% when farming a block

AmineKhaldi committed:
Addendum to 100% block fill rate. (#19006)

emlowe commented on chia/_tests/core/mempool/test_mempool_manager.py:
```diff
@@ -1081,7 +1081,7 @@ async def make_and_send_big_cost_sb(coin: Coin) -> None:
         sig = AugSchemeMPL.sign(sk, IDENTITY_PUZZLE_HASH, g1)
         aggsig = G2Element()
-        for _ in range(169):
+        for _ in range(242):
```
do you want to comment what this magic number is

arvidn replied:
yeah, it's not great. it's the number of transactions we need to fill the mempool sufficiently to trigger the expected number of rejections. going from fill rate 70% -> 100%, I simply took 169 / 0.7 to increase this number proportionally. I'll try to improve this a bit.

AmineKhaldi replied:
Sent https://github.com/Chia-Network/chia-blockchain/pull/19026 as an attempt to clarify this.

emlowe replied:
I think I might prefer the simpler construct with a comment like Amine has above. The latest version here Arvid has more magic numbers :-)

arvidn replied:
I think magic numbers are "more magic" the harder they are to explain. And breaking this down to how the magic number is computed makes the new ones less magical. (and I think in the most recent update there no magic numbers). As long as you accept that the CLVM cost of returning a quoted list is 44 and that it takes 88 bytes to serialize another agg-sig condition.

There are a lot of mempool tests with magic number that need tweaking when tweaking constants. Now we're moving this test to have its expectations properly rooted in the test constants it's using. I think this really helps humans understand (and gain confidence in) the test. I will follow up (in a separate PR) to make sure this test

arvidn committed:
break down 'magic number' in test into its components

AmineKhaldi committed:
Attempt to clarify why we add many aggsig conditions in make_and_send_big_cost_sb (#19026)

Attempt to clarify why we add many aggsig conditions in make_and_send_big_cost_sb.
